tests/features/coordinator.feature

```diff
@@ -47,6 +47,7 @@ Feature: Interacting with coordinator
           - client_hostname: pgconsul_postgresql3_1.pgconsul_pgconsul_net
             state: streaming
         """
+        Then container "postgresql2" replication state is "sync"
```

Comment:
`postgresql2` is not primary in this scenario, but `replication state is` check is for primary, it checks primary synchronous_standby_names: https://github.com/yandex/pgconsul/blob/main/tests/steps/database.py#L91
You should check `postgresql1` container here.